Vision/Vision.bs

```diff
@@ -86,16 +86,16 @@ Markup Shorthands: markdown yes
 	and its impact will continue to grow in the future, 
 	as it expands reach, knowledge, education, and services even more broadly.
 	We believe the World Wide Web should be 
-	inclusive and respectful of those who use it: 
+	inclusive and respectful of everyone:
```

Comment:
I have a problem with "everyone".  As phrased, this raises the paradox of being tolerant of the intolerant.
```suggestion
	inclusive and respectful of all participants:
```

Reply:
WFM

Reply:
Both "everyone" and "all participants" feel as though they miss the mark of the intention in this section. Considering the comments made in #211 and the substance of the EWP we refer to right before this section, I wonder if this is a good place to allude to both our commitment to humanity and to life on this planet (through sustainability).

Something like "We believe the World Wide Web should be respectful and inclusive of life on earth:"

Alternately, something like "...those who interact with it:" (to avoid the "use" issue for those who don't actively engage).

Reply:
"life on earth" is overreaching by far.  We do not, in fact, include amoebas, or deep-sea vent worms.  This overreaching makes the phrase nearly comical.  I dislike this, for the same reasons I dislike everyone, but the only remaining suggestion I have is "all".

```suggestion
	inclusive and respectful of all:
```